### DISCUSSION TITLE
голос при переводе с японского

### DISCUSSION THREAD

**Discussion by giperforg** [ilyhalight/voice-over-translation#949] [Q&A] [answered]
здравствуйте
у меня при переводе с японсого нет голосов(субтитры есть)
при переводе с английского голоса есть
и ещёб можно ли как то добавить свой сайт в скрипт что бы на нем работал перевод?

**Comment by ilyhalight** [comment, marked as answer]
> у меня при переводе с японсого нет голосов(субтитры есть)
> при переводе с английского голоса есть

Значит, что-то пошло не так на стороне яндекса, на это повлиять я не могу

>  и ещёб можно ли как то добавить свой сайт в скрипт что бы на нем работал перевод?

Делайте форк и прописывайте нужный функционал. Какую-нибудь универсальную инструкцию тут не дать
